rr_rviz_plugins/src/ChassisPanel.cpp

```diff
@@ -0,0 +1,198 @@
+#include <pluginlib/class_list_macros.h>
+#include <rr_msgs/chassis_state.h>
+#include <rr_rviz_plugins/ChassisPanel.h>
+#include <std_msgs/Bool.h>
+#include <QVBoxLayout>
+#include <QtGui/QPaintEvent>
+#include <QtGui/QPainter>
+#include <QtWidgets/QLabel>
+#include <cmath>
+
+namespace rr_rviz_plugins {
+bool received_signal = false;
+int cur_speed;
+float hding;
+bool record_bag;
+bool estop;
+QLabel *speed_label;
+QLabel *heading_label;
+QLabel *msg_label;
+QLabel *rcding_bag_label;
+QLabel *estop_label;
+ros::Time now;
+ros::Time chassisUpdateTime = ros::Time::now();
+ros::Timer time;
+QString spacing = "          ";
```

Comment:
Probably should be const. Not the biggest fan of just adding spacing, maybe should try to shift the label itself by setting its position.